typescript_tests/TrialTable.test.tsx

```diff
@@ -124,7 +126,11 @@ const studyDetail: StudyDetail = {
 
 it("Sort TrialTable by trial number", () => {
   const { getAllByRole, getByText } = render(
-    <TrialTable studyDetail={studyDetail} isBeta={false} />
+    <RecoilRoot>
+      <SnackbarProvider>
```

Comment:
Just a question, if we remove these two components here, will test cases fail?

Reply:
Yes. In the TrialTable, we started using recoil and snackbar because we call a `action.tellTrial` inside. That is the reason I need to wrap it with `<RecoilRoot>` and `<SnackbarProvider>`.